.github/workflows/dependency_merge.yml

```diff
@@ -1,16 +1,16 @@
-name: Dependabot Auto Merge
-on: pull_request
-
-permissions:
-  pull-requests: write
+name: Dependabot Auto Merge - ZStream
+on:
+  pull_request_target:
+    types: ["opened", "synchronize", "reopened"]
+    branches-ignore:
+      - master
 
 jobs:
   dependabot:
     name: dependabot-auto-merge
     runs-on: ubuntu-latest
     if: |
-      github.event.pull_request.user.login == 'Satellite-QE' &&
-      contains( github.event.pull_request.labels.*.name, 'dependencies')
+      contains(github.event.pull_request.labels.*.name, 'dependencies')
```

Comment:
if we're going to check the `dependencies` label on auto-cherrypicked PRs for auto-merge, then we need an additional task to add this label if main PR also contains this, right?

Reply:
yes, I get you this isn't for master, but we're talking about the dependancy automerge for zstream, but we miss `dependancy` and `automerge-cherrypicked` labels on the autocherrypicked PRs, which we've merging manually https://github.com/SatelliteQE/robottelo/pull/12615

Reply:
@Gauravtalreja1 Nice Catch! At least by looking at GHA `automerge-cherrypicked` label is not required for cherrypicked dependency PRs to merge but `dependency` is must.

@omkarkhatavkar I think missing that label in cherrypicked dependency PRs is the cause of aborted AutoMerges in zStream versions!

Reply:
@jyejare even after I was adding label it was failing

Reply:
@omkarkhatavkar Probably because at that time we did not have this fix. Now we should fix it by:

- Adding dependency label to Dependabot PRs which are cherrypicked by `ACP GHA`!

Reply:
@Gauravtalreja1 @omkarkhatavkar PR open to fix missing dependencies label in zStream PRs: https://github.com/SatelliteQE/robottelo/pull/12682

Reply:
@Gauravtalreja1 Since the PR #12682 is now merged , can we merge this as well ! We can wait for cherrypicks to merge though !